GitFyle.Core.Api/Brokers/Storages/StorageBroker.Configurations.Configuration.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.Configurations;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        private void AddConfigurationConfigurations(EntityTypeBuilder<Configuration> builder)
+        {
+            builder
```

Comment:
Can you remove this restriction of 255 on Name Value and Type.

Reply:
I did as you asked